v1-scripts/configs/ZenXY/Ramps_16T_LCD_32

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+  
+incdir=$(dirname $0)/..
+
+restore_configs
+
+. $incdir/base-zenxy-config
+. $incdir/drv8825
+
+opt_set DEFAULT_AXIS_STEPS_PER_UNIT "{ 200, 200, 800, 200 }"
+opt_enable REVERSE_ENCODER_DIRECTION
```

Comment:
I think reverse encoder direction should probably be the same on all the LCD configs. I'm guessing there is an error in the configs that causes this to only be on some setups.

Reply:
This probably should be put next to the disolay enable feature, which looks like it is in the base configs.